### PR TITLE
Add related-tooling.md

### PR DESCRIPTION
**Pull request by mikedep333**
Currently lists:
- Foreman/Katello
- Ansible Modules for Pulp aka Squeezer
- pulp_rpm_repos

### CODE REVIEW

**Comment by mikedep333** [comment]
@juan-cabrera @mdellweg Your Ansible roles/modules are listed here.

File: related-tooling.md
Comment (mdellweg) [inline related-tooling.md]: :champagne:
I know, i wrote this text, but i think, we should leave the aim to the project page.
Here we could skip the "complete" fore now and add a note that file related modules are already available.

Comment (mikedep333) [reply related-tooling.md]: Updated, thanks!

**Comment by mikedep333** [comment]
@fpytloun I added docker-pulp as well.

**Comment by mikedep333** [comment]
Note that this change adds the "Related Tooling" to both the top of the site and the side.

File: related-tooling.md
Comment (CodeHeeler) [inline related-tooling.md]: "managing repositories(['] or [of] added here) software packages"

Also, there is mixed use of "and" and "&" throughout, perhaps pick one and stay consistent?

File: related-tooling.md
Comment (CodeHeeler) [inline related-tooling.md]: "wildly inspired"? maybe widely or largely?

File: related-tooling.md
Comment (CodeHeeler) [inline related-tooling.md:26]: This is not specific to your PR, as we all could do better, so if you'd prefer to open an additional issue, I understand, but the interchangeable use of "pulp" and "Pulp" should probably be replaced with one consistent use.  (In the first line of the pulp_rpm_repos section, it's "pulp server" and in the following line, it's "Pulp 3 server".

File: _data/topnav.yml
Comment (CodeHeeler) [inline _data/topnav.yml:18]: Just a note that we are now adding even more material to an already crowded bar, perhaps we should move this issue (https://github.com/pulp/pulpproject.org/issues/182) to consolidate some/all into a dropdown menu instead should move up in priority so as not to have a bad user experience on smaller screens/mobile devices?

**Review by CodeHeeler — APPROVED** [review]
Comments added, mostly minor, none blocking, thanks for working on this!

File: related-tooling.md
Comment (mdellweg) [reply related-tooling.md]: Actually, i think this last sentence does not carry any useful information for the user either.
It is part of the repo description, because i was a bit lazy to write better documentation for developers.

File: related-tooling.md
Comment (mikedep333) [reply related-tooling.md:26]: I agree and will do a find and replace with "Pulp", like our official docs use. I personally wrote "Pulp", but I copied and pasted these descriptions with as few changes as possible.

Perhaps I should use markdown quote blocks instead?

File: related-tooling.md
Comment (mikedep333) [reply related-tooling.md]: As a user, I would find the fact that it complements another project very useful. But I will defer to the author's judgement ;)

File: related-tooling.md
Comment (CodeHeeler) [reply related-tooling.md:26]: If you are wholesale taking descriptions from elsewhere, sure, quote blocks would make sense.  Also ok with a find replace, your choice.

File: related-tooling.md
Comment (mikedep333) [reply related-tooling.md]: I'll fix that "managing repositories software packages" typo.

I wrote the 2nd paragraph, but copied & pasted the 1st.

I'll replace "&" with "and".

As my own personal style guide, I like to use "&" for words that commonly go together ("peanut butter & jelly"), but use "and" for everything else ("I went to the store and bought tea.") However, this page is not my own personal document.

File: _data/topnav.yml
Comment (mikedep333) [reply _data/topnav.yml:18]: Yeah, I'm wondering about this too.

As much as I hate delaying a merge, it seems like Brian did the last big restructuring of the site a0fcf28d4654afed635c05ff42ef44eed8e5fa8e https://pulp.plan.io/issues/1862 . So let's wait for his feedback.

Comment (CodeHeeler) [reply _data/topnav.yml:18]: I'd be ok w/ not delaying merge as this has an issue to track the work separately already, it's just that it is on github and not plan.io and has a tendency to get forgotten, so we would want to make sure it stays on the radar.

File: related-tooling.md
Comment (mdellweg) [reply related-tooling.md]: Most of the original text is a placeholder anyway. Maybe it is about time to rethink that readme file.
Given that nothing here is carved in stone, i am fine with either.

**Comment by juan-cabrera** [comment]
My role has not been updated since Pulp 3.0.0rc2. I can't guarantee that it works with the current version.

**Comment by mikedep333** [comment]
> My role has not been updated since Pulp 3.0.0rc2. I can't guarantee that it works with the current version.

@juan-cabrera  I assumed this, but I still want to list it anyway.

Chronically, many of the projects on this page will not work with the latest version. But we list them here not only so that users can find & use them, but also so that they will contribute to them (of if need be, fork them.)

Are you still OK with me listing it?

File: _data/topnav.yml
Comment (mikedep333) [reply _data/topnav.yml:18]: @CodeHeeler I agree. Please re-review in the meantime, I'm waiting for juan-cabrera to reply also.

**Review by CodeHeeler — APPROVED** [review]
Looks good, thanks!

**Comment by juan-cabrera** [comment]
> > My role has not been updated since Pulp 3.0.0rc2. I can't guarantee that it works with the current version.
> 
> @juan-cabrera I assumed this, but I still want to list it anyway.
> 
> Chronically, many of the projects on this page will not work with the latest version. But we list them here not only so that users can find & use them, but also so that they will contribute to them (of if need be, fork them.)
> 
> Are you still OK with me listing it?

Yes, hopping this can help or inspire someone

**Comment by mikedep333** [comment]
@CodeHeeler I lack perms to merge; please merge now.

I will put https://github.com/pulp/pulpproject.org/issues/182 on my TODO list.

**Pull request merged**
